### PR TITLE
Add exercise: transpose

### PR DESCRIPTION
**Pull request by andrej-makarov-skrt**

### CODE REVIEW

File: exercises/transpose/example.jl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    result = fill("", rows)
+    for i = 1:rows
+        for j = 1:cols
+            sym = ""
+            if i <= length(input[j])
+                delta = j-length(result[i])-1
+                if delta > 0
+                    sym = " " ^ delta
+                end
+                sym = sym * string(input[j][i])
```
Comment (SaschaMann) [inline exercises/transpose/example.jl]: `sym *= string(...)` would work as well, looks a bit cleaner imo but that's personal taste.

**Comment by SaschaMann** [comment]
Looks good! Thanks for adding so many exercises :)

**Comment by aawray** [comment]
So that is what I do just using regular expressions parsing JSON file. :)

File: exercises/transpose/example.jl (hunk 1 above)
Comment (aawray) [inline exercises/transpose/example.jl]: Yes, thank you!

**Comment by SaschaMann** [comment]
Could you upload/share that script? Would be very useful to have.

**Comment by andrej-makarov-skrt** [comment]
Here's Python script that must be changed per exercise: https://gist.github.com/andrej-makarov-skrt/492f133372e8c3d42c5cbad168253375

I use regexp in Atom editor for fast string replacements.

**Comment by SaschaMann** [comment]
Thanks!

**Comment by andrej-makarov-skrt** [comment]
@SaschaMann updated the script, I used it for atbash-cipher exercise, so you can check it out as an extended example.

**Pull request merged**